content/blog/learn-what-you-can-do-with-hpe-data-services-cloud-console-api-in-just-3-minutes.md

```diff
@@ -0,0 +1,201 @@
+
+Alternatively, you can export the public collection locally to your local storage as JSON file and import it as a new collection in your Postman workspace. 
+
+> **Note:** You must sign in to your Postman account to create a fork or export the public collection. To fork a collection within a public workspace, you must also enable your public profile in your Postman profile settings.  
+
+### Step 3 – Set your Data Services Cloud Console collection variables
+
+The Data Services Cloud Console API collection built by Mark makes use of collection variables that are available throughout the REST API requests in the collection. Select the collection in your workspace and then select the ***Variables*** tab as shown below.
+
+![HPE Data Services Cloud Console collection variables](/img/variables-dscc-collection-postman-figure2.png "HPE Data Services Cloud Console collection variables")
+
+<span style="color:grey; font-family:Arial; font-size:1em">Figure 2: HPE Data Services Cloud Console collection variables.</span>
+
+Define the Current Value of the collection variables to match your Data Services Cloud Console context:    
+
+* **baseUrl**: This variable defines the base URL of the REST API requests. It should match the regional endpoint of your Data Services Cloud Console application instance where your storage devices are registered.    
+
+* **ClientId** and **ClientSecret**: they should be set with the value of your Client Application API credentials you previously created using the HPE GreenLake Cloud Platform GUI. These variables are used to request an OAuth access token by authenticating with the authorization server referenced in the **sso_URI** variable.    
+
+* **sso_URI**: This variable is the URI of the OAuth authorization server. If your organization has set up their own HPE GreenLake SAML Single Sign-On (SSO) authorization server to create access token, replace the current default value with your SSO URI. Otherwise keep the value for this variable as currently set to *sso.common.cloud.hpe.com/as/token.oauth2*.    
+
+* **BearerToken:** Do not edit this variable. Keep the value field empty. The collection variable BearerToken will be set automatically upon successful execution of the ***GetToken*** API call as explained in the next step.    
+
+
+### Step 4 – Acquire an OAuth access token as your session bearer token
+
+Data Services Cloud Console API uses bearer token as authorization type to ensure that all REST API requests access authorized data services securely. So you first need to obtain a token from the OAuth authorization server before you can make any REST API calls to your regional Data Services Cloud Console application instance. To do so, proceed as follows:
```

Comment:
I feel as if you may have cut and paste from some documentation written by Ron Dharma here... There are a couple of articles missing in the first sentence. I'm always finding that as an issue with him and I can't always fix his because there tend to be so many errors in this regard.

The first sentence should read "...uses **a** bearer token as **an** authorization type..."

Reply:
That would be really weird because in French the articles are so completely associated with the noun - and very particular in terms of masculine, feminine, or plural!